.travis.yml

```diff
@@ -48,7 +53,11 @@ deploy:
     repo: nokia/git-changelog-generator
 - provider: script
   skip_cleanup: true
-  script: bash scripts/bintray/deploy.bash "$BINTRAY_USER" "$BINTRAY_KEY" "$(< version.txt)"
+  script: >
+            DEBUG=1 bash scripts/bintray/deploy.bash -u "$BINTRAY_USER" \
```

Comment:
You shouldn't need the '\' at the end of the lines, because the yaml multiline string should already handle that, but I think you have to use the same level of indentation for all the lines. Feel free to leave it as it is.

Reply:
I didn't know that! 
Humm.. I'll give it a try. I'd rather they let me choose to indent or not, but hey hum - better that than the trailing backslashes